src/main/frontend/components/publishing.cljs

```diff
@@ -0,0 +1,189 @@
+(ns frontend.components.publishing
+  (:require [rum.core :as rum]
+            [frontend.context.i18n :as i18n]
+            [frontend.db :as db]
+            [frontend.state :as state]
+            [frontend.util :as util]
+            [reitit.frontend.easy :as rfe]
+            [frontend.db-mixins :as db-mixins]
+            [frontend.config :as config]
+            [lambdaisland.glogi :as log]
+            [promesa.core :as p]
+            [frontend.handler.page :as page-handler]
+            [frontend.handler.notification :as notification]
+            [clojure.string :as string]
+            [frontend.ui :as ui]
+            [frontend.components.svg :as svg]
+            [frontend.handler.project :as project-handler]))
+
+(defn get-published-pages
```

Comment:
Db fns should be separated from the view components.

Reply:
Will move this into `db.clj`.